libs/@local/hash-backend-utils/src/service-usage.ts

```diff
@@ -258,7 +262,7 @@ export const createUsageRecord = async (
       relation: "administrator",
       subject: {
         kind: "accountGroup",
-        subjectId: hashInstanceAdminGroupId,
+        subjectId: hashInstanceAdminGroup.id,
         subjectSet: "member",
       },
```

Comment:
How / where is this preserved?

Reply:
The instance admin group is known to the graph, so it's encoded there:
https://github.com/hashintel/hash/blob/314207d48a47e514957e35a32b778a183bdda91b/libs/%40local/graph/postgres-store/src/store/postgres/seed_policies.rs#L267-L296

Reply:
Ah okay, I think `recordsUsageOf` needs adding to that list then

Reply:
Applied in [`d20332e` (#7254)](https://github.com/hashintel/hash/pull/7254/commits/d20332ee55af5dff462e3a26ffb80ad136b5c340)